.github/workflows/codeql-analizer.yml

```diff
@@ -0,0 +1,35 @@
+name: "Code Scanning - Action"
+
+on:
```

Comment:
same here., we are going to run code analysers on our own infra!

Reply:
That does not provide the public with its result